proto/tests/server/test_no_pipeline_config.cpp

```diff
@@ -80,6 +80,7 @@ class TestNoForwardingPipeline : public ::testing::Test {
 constexpr char TestNoForwardingPipeline::grpc_server_addr[];
 
 TEST_F(TestNoForwardingPipeline, Write) {
+  EXPECT_EQ(PIGrpcServerGetPort(), 50051);
```

Comment:
Thanks for adding a test but please write a separate test case for this.
Adding this in an existing test case in this file doesn't make sense IMO. Maybe create a new file for that kind of generic server test (test_server.cpp?).

Reply:
Moved test to new file test_pi_server.cpp